### PR TITLE
v1.8: Postman

### LINKED ISSUE #2793: v1.8: Update Postman
Update postman collection to reflect v1.8 updates.

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2793

### CODE REVIEW

File: assets/misc/meilisearch-collection-postman.json [hunk 1]
```diff
@@ -2184,6 +2184,81 @@
...
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "150"
+								},
+								"url": {
+									"raw": "{{url}}/indexes/{{indexUID}}/settings/proximity-precision",
```
Comment (curquiza) [inline assets/misc/meilisearch-collection-postman.json:2229]: not proximityprecision

File: assets/misc/meilisearch-collection-postman.json [hunk 2]
```diff
@@ -2184,6 +2184,81 @@
...
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"indexes",
+										"{{indexUID}}",
+										"settings",
+										"proximity-precision"
```
Comment (curquiza) [inline assets/misc/meilisearch-collection-postman.json:2237]: not proximityprecision

File: assets/misc/meilisearch-collection-postman.json [hunk 3]
```diff
@@ -2184,6 +2184,81 @@
...
+							},
+							"response": []
+						},
+						{
+							"name": "Reset search cut-off",
+							"request": {
+								"method": "DELETE",
+								"header": [],
+								"url": {
+									"raw": "{{url}}/indexes/{{indexUID}}/settings/proximity-precision",
```
Comment (curquiza) [inline assets/misc/meilisearch-collection-postman.json:2249]: not proximityprecision

File: assets/misc/meilisearch-collection-postman.json [hunk 4]
```diff
@@ -2184,6 +2184,81 @@
...
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"indexes",
+										"{{indexUID}}",
+										"settings",
+										"proximity-precision"
```
Comment (curquiza) [inline assets/misc/meilisearch-collection-postman.json:2257]: not proximityprecision

**Review by curquiza — CHANGES_REQUESTED** [review]
You did not change the version 😇

**Comment by guimachiavelli** [comment]
🤦 

Postman didn't actually save most of my changes. Sorry, I should've checked the files tab instead of assuming the export worked as expected.

**Review by curquiza — CHANGES_REQUESTED** [review]
it's search-cutoff-ms, not search-cut-off-ms

You can test it with rc1 who is already available 😉

**Review by curquiza — APPROVED** [review]

**Pull request merged**